data/presets/amenity/vending_machine/bicycle_tube.json

```
@@ -0,0 +1,21 @@
+{
+    "icon": "temaki-vending_machine",
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "inner tube",
+        "bicycle tubes",
+        "bike tube"
+    ],
+    "tags": {
+        "amenity": "vending_machine",
+        "vending": "bicycle_tube"
+    },
+    "reference": {
+        "key": "vending",
+        "value": "bicycle_tube"
+    },
+    "name": "Bicycle inner tube Vending Machine"
```

Please use title case for the preset name

```suggestion
    "name": "Bicycle Inner Tube Vending Machine"
```